[Bug]: unit test apache_beam/coders/row_coder_test.py::RowCoderTest::test_row_coder_fail_early_bad_schema fails locally

### What happened?

test_row_coder_fail_early_bad_schema fails when run test suites in non-parallel way:
```
$ pytest -v apache_beam/coders/row_coder_test.py::RowCoderTest

apache_beam/coders/row_coder_test.py::RowCoderTest::test_row_coder_fail_early_bad_schema FAILED                                                             [ 60%]

E     AssertionError: "type_with_no_typeinfo" does not match "Encountered a type that is not currently supported by RowCoder: "
```
but successes when run with pytes -n flag, or run independently:
```
$ pytest -v -n 2 apache_beam/coders/row_coder_test.py::RowCoderTest

[gw0] [ 53%] PASSED apache_beam/coders/row_coder_test.py::RowCoderTest::test_row_coder_cloud_object_schema 
apache_beam/coders/row_coder_test.py::RowCoderTest::test_row_coder_fail_early_bad_schema
```
or
```
$ pytest -v apache_beam/coders/row_coder_test.py::RowCoderTest::test_row_coder_fail_early_bad_schema

apache_beam/coders/row_coder_test.py::RowCoderTest::test_row_coder_fail_early_bad_schema PASSED                                                             [100%]
```

Reproduced with python3.8 on both macOS and windows 10.


### Issue Priority

Priority: 3

### Issue Component

Component: sdk-py-core

--- a/sdks/python/apache_beam/coders/row_coder_test.py
+++ b/sdks/python/apache_beam/coders/row_coder_test.py
@@ -369,14 +369,15 @@ class RowCoderTest(unittest.TestCase):
         fields=[
             schema_pb2.Field(
                 name="type_with_no_typeinfo", type=schema_pb2.FieldType())
-        ])
+        ],
+        id='bad-schema')
 
     # Should raise an exception referencing the problem field
     self.assertRaisesRegex(
         ValueError, "type_with_no_typeinfo", lambda: RowCoder(schema_proto))
 
   def test_row_coder_cloud_object_schema(self):
-    schema_proto = schema_pb2.Schema()
+    schema_proto = schema_pb2.Schema(id='some-cloud-object-schema')
     schema_proto_json = json_format.MessageToJson(schema_proto).encode('utf-8')
 
     coder = RowCoder(schema_proto)

